Test/Utils/ContrailAPI_New/Models/DNSServer.ps1

```diff
@@ -22,8 +22,8 @@ class DNSServer : BaseResourceModel {
         $this.Dependencies += [Dependency]::new('virtual-DNS-record', 'virtual_DNS_records')
     }
 
-    [String[]] GetFqName() {
-        return @($this.DomainName, $this.Name)
+    [FqName] GetFqName() {
+        return [FqName]::New(@($this.DomainName, $this.Name))
```

Comment:
I left DomainName, because it is also used in line 31. So I could either do it like that or store domain name in ParentFqName, but then I would have to do something like $ParentFqName.ToStringArray()[0] to get domain name. 
I believe solution that I have chosen is more clear.